docs/cue/module/signing.md

```diff
@@ -91,6 +91,29 @@ cosign verify ghcr.io/my-org/modules/my-app:1.0.0 \
   --certificate-oidc-issuer-regexp=<your issuer URL>
 ```
 
+!!! tip "Verify Signature from GitHub action or GitLab CI/CD"
+
+    When the signature was created in GitHub via the keyless signing you should set the flag `--certificate-identity-regexp`
+    to a value like `^https://github.com/<user|org>/<repo-name>.*` and set the flag `--certificate-oidc-issuer-regexp`
+    to `https://token.actions.githubusercontent.com`.
+
+    GitLab is the same setup for the identity flag, `https://gitlab.com` instead of `https://github.com` and the oidc
+    issuer is the GitLab instance domain e.g. `https://gitlab.com`
+
+Example verification of the podinfo module:
+
+```shell
+cosign verify ghcr.io/stefanprodan/modules/podinfo:latest --certificate-identity-regexp="^https://github.com/stefanprodan/podinfo.*$" --certificate-oidc-issuer=https://token.actions.githubusercontent.com
+
```

Comment:
```suggestion
$ cosign verify ghcr.io/stefanprodan/modules/podinfo:latest \
--certificate-identity-regexp="^https://github.com/stefanprodan/podinfo.*$" \
--certificate-oidc-issuer=https://token.actions.githubusercontent.com
```

Reply:
I left out the `$` as it's not in the other shell script examples, but I guess because there is output here it helps distinguish the command from the output?

Reply:
Ah yes, let's remove the output and leave only the command so people can copy/paste also no `$`

Reply:
Grand, will update it :+1: